src/backend/functions/src/system/fakeDataPopulator.ts

```diff
@@ -70,17 +77,16 @@ export class FakeDataPopulator {
         'price': faker.datatype.number(8999),
       };
 
-      await this.createMerchantProduct(merchatId, product);
+      await this.createMerchantProductForSpecificRegion(merchantId, product);
     }
   }
-
-  private async createMerchantProduct(merchantId: string, product: any) {
-    await this.firestoreDatabase.collection('merchants').doc(merchantId).collection('products').add(product);
+  private async createMerchantDocumentForSpecificRegion(merchant: any, regionId: string): Promise<string> {
+    let documentReference = await this.firestoreDatabase.collection('regions').doc(regionId).collection('merchants').add(merchant);
```

Comment:
Lets move `merchants` into a constant as well since it's used in multiple places.